funnel/templates/notifications/layout_email.html.jinja2

```diff
@@ -1,45 +1,511 @@
 <!DOCTYPE html>
-<html lang="{{ get_locale() }}">
-  <head>
+<html lang="{{ get_locale() }}" xmlns="http://www.w3.org/1999/xhtml" xmlns:v="urn:schemas-microsoft-com:vml" xmlns:o="urn:schemas-microsoft-com:office:office">
```

Comment:
What are the `xmlns` attrs for?

Reply:
These are for the ``vml`` elements that render in outlook.